apps/meteor/tests/end-to-end/api/livechat/11-livechat.ts

```diff
@@ -75,6 +77,33 @@ describe('LIVECHAT - Utils', function () {
 			expect(body.config).to.have.property('settings');
 			expect(body.config).to.have.property('departments').that.is.an('array');
 		});
+		it('should have custom fields data', async () => {
+			const customFieldName = `new_custom_field_${Date.now()}`;
+			await createCustomField({
+				searchable: true,
+				field: customFieldName,
+				label: customFieldName,
+				defaultValue: 'test_default_address',
+				scope: 'visitor',
+				visibility: 'visible',
+				regexp: '',
+				public: true,
+				required: false,
+
+				options: '',
+			} as unknown as ILivechatCustomField & { field: string });
```

Comment:
I've found `createCustomField` being used in 5 different places, and all of them have this type cast.. it doesn't feel correct to me..

why don't you fix `createCustomField` signature instead of having to using this type cast in every place?